File: Modul 6/Python/PRAK605-2410817120018-ArindaDitaPratiwi.py
n = int(input())
matriks1 = []; matriks2 = []; matriksAXB = []
print('Matriks A')
for x in range(n):
    z =  list(map(int, input().split()))
    matriks1.append(z)
print('Matriks B')
for x in range(n):
    z =  list(map(int, input().split()))
    matriks2.append(z)
print('Matriks AXB')
for x in range(n):
    matriksAXB.append([])
    for y in range(n):
        jumlah = 0
        for z in range(n):
            jumlah = jumlah + matriks1[x][z] * matriks2[z][y]
        matriksAXB[x].append(jumlah)
for isi in matriksAXB:
    for z in isi:
        print(z, end = " ")
    print()

File: Modul 6/C/PRAK605-2410817120018-ArindaDitaPratiwi.c
#include <stdio.h>
int main(){
    int x, y, z, n, jumlah = 0;
    int matriks1[10][10], matriks2[10][10], hasil[10][10];
    scanf("%d", &n);
    printf("Matriks A\n");
    for(x = 0; x < n; x++){
        for (y = 0; y < n; y++){
            scanf("%d", &matriks1[x][y]);}}
    printf("Matriks B\n");
    for(x = 0; x < n; x++){
        for (y = 0; y < n; y++){
            scanf("%d", &matriks2[x][y]);}}
    for (x = 0; x < n; x++){
        for (y = 0; y < n; y++){
            for (z = 0; z < n; z++){
                jumlah = jumlah + matriks1[x][z] * matriks2[z][y];}
            hasil[x][y] = jumlah;
            jumlah = 0;}}
    printf("Matriks AXB\n");
    for (x = 0; x < n; x++){
        for (y = 0; y < n; y++){
            printf("%d ", hasil[x][y]);}
        printf("\n");}
    return 0;}